pkg/virtctl/create/vm/vm_test.go

```diff
@@ -575,14 +590,17 @@ chpasswd: { expire: False }`
 			vm, err := decodeVM(out)
 			Expect(err).ToNot(HaveOccurred())
 
-			Expect(vm.Spec.Template.Spec.Volumes).To(HaveLen(2))
-			Expect(vm.Spec.DataVolumeTemplates[0].Spec.Storage.Resources.Requests[k8sv1.ResourceStorage]).To(Equal(resource.MustParse(size)))
+			Expect(vm.Spec.DataVolumeTemplates).To(HaveLen(2))
```

Comment:
You are replacing here the `Volumes` wit the `DataVolumeTemplates`, is this correct?

Reply:
The assertion of `Volumes` was moved to L601. The assertion of `DataVolumeTemplates` was missing before.